gtda/diagrams/features.py

```diff
@@ -256,6 +258,28 @@ def transform(self, X, y=None):
             transpose((1, 0, 2))
         return Xt
 
+    def plot(self, Xt, sample=0, **layout):
+        """Plot betti curves, one per homology dimensions. If :param:`sample`
+        is a list, betti surfaces are plotted.
+
+        Parameters
+        ----------
+        Xt : ndarray of shape (n_samples, n_homology_dimensions, n_bins)
+            Betti curves: as returned by :meth:`transform`.
+            One curve (represented as a one-dimensional array
+            of integer values) per sample and per homology dimension seen
+            in :meth:`fit`. Index i along axis 1 corresponds to the i-th
+            homology dimension in :attr:`homology_dimensions_`.
+        sample : int or list of int, optional, default: ``0``
+            Index(/indices) of the sample(s) to be plotted. If multiple indices
+            are provided, a betti surface using those is plotted.
+        """
+        if isinstance(sample, int):
+            sample = [sample]
+        return plot_betti_surfaces(
```

Comment:
Since we have a Betti *curve* plotting function, I am surprised to see the betti surface one being used here.

Reply:
betti_surface falls back to betti curve,if there is only one sample to plot

Reply:
It seems you are trying to go beyond the proposal of plotting individual samples in this PR. I think this is too much at this stage.

Reply:
That is to say, if we were serious about this, we should intervene directly on `PlotterMixin` and discuss what we want the new mixin to support.

Reply:
ok, going back to `plot_betti_curve`